Makefile

```diff
@@ -1,4 +1,4 @@
-SUBDIRS ?= awscodecommit awscognito awsdynamodb awskinesis awssqs
+SUBDIRS ?= cmd/awscodecommitsource awscognito awsdynamodb awskinesis awssqs
```

Comment:
Will this change improve the test scenario? 

```diff
diff --git a/cmd/awscodecommitsource/Makefile b/cmd/awscodecommitsource/Makefile
index f37bf66..0617a03 100644
--- a/cmd/awscodecommitsource/Makefile
+++ b/cmd/awscodecommitsource/Makefile
@@ -1,4 +1,6 @@
 PACKAGE = awscodecommitsource
 PACKAGE_DESC = Triggermesh AWS CodeCommit Event Source
 
+GOPKGS = ../../pkg/...
+
 include ../../scripts/inc.Makefile
diff --git a/scripts/inc.Makefile b/scripts/inc.Makefile
index 88b65f4..4eb5bb6 100644
--- a/scripts/inc.Makefile
+++ b/scripts/inc.Makefile
@@ -14,6 +14,8 @@ GOFMT      ?= gofmt
 GOLINT     ?= golint
 GOTOOL     ?= go tool
 GOTEST     ?= gotestsum --junitfile $(OUTPUT_DIR)$(PACKAGE)-unit-tests.xml --
+
+GOPKGS     +=
 LDFLAGS    +=
 
 .PHONY: help mod-download build install release test coverage lint vet fmt fmt-test image clean
@@ -44,7 +46,7 @@ release: ## Build release binaries
  done
 
 test: ## Run unit tests
- $(GOTEST) -cover -coverprofile=c.out ./...
+ $(GOTEST) -cover -coverprofile=c.out $(GOPKGS) ./...
 
 coverage: ## Generate code coverage
  $(GOTOOL) cover -html=c.out -o $(OUTPUT_DIR)$(PACKAGE)-coverage.html
```

Reply:
It may yes, but if each source's test goal runs tests for the entire pkg/, maybe we'd be better of having a top level test goal instead of one per source to avoid running those tests n times?

Reply:
completely agree. 
we can also specify the specific pkg we want to test if that's preferred

Reply:
Done in 6cfff42d5460eea5bde65236392cf9fc06d0033e